### ISSUE TITLE
[Platform Bug] Report 27 "Dimensions - Total"

### ISSUE THREAD

**Issue by vlkov** [microsoft/ALAppExtensions#22563]
Hello,

The trigger OnOpenPage() was called twice. First when the request page was open, second when user run the report preview.
So, if user change the dimension selection the report restore default selection and it cause the error behaviors.

If use sent to pdf option, report work corrects.

Thank you!

**Comment by pri-kise** [comment]
I guess this is related to the following platform change: Multiple Preview of Reports.
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-report-triggers?tabs=current%2Crdlc#modes

The fast work-around could be to set AllowScheduling to false (https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-allowscheduling-property)

Maybe the trigger `OnAfterGetCurrRecord` can be used instead, since this trigger seems to called only once for the master instance

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 10731.
